### ISSUE TITLE
JSON support

### ISSUE THREAD

**Issue by alex-tan** [winterland1989/mysql-haskell#21]
Was taking a glance at this as mysql-simple doesn't support the JSON column. It didn't look like this project has support for that either yet, am I correct about that? Thanks.

**Comment by winterland1989** [comment]
Yes, the `MySQLValue` type have no native support for JSON column yet.
